stanford/roles/cluster/defaults/main.yml

```diff
@@ -90,6 +95,19 @@ CLUSTER_SUBNET: "{{ CLUSTER_SUBNETS[CLUSTER_NAME][deployment_subnet_index|int] }
 
 CLUSTER_SECURITY_GROUP_RULES_INGRESS: "{{ CLUSTER_SECURITY_GROUP_RULES[CLUSTER_NAME] }}"
 CLUSTER_SECURITY_GROUP_RULES:
+  analytics-jenkins:
+    - proto: tcp
+      from_port: 22
+      to_port: 22
+      cidr_ip: "{{ DEPLOYMENT_CIDR }}"
+    - proto: tcp
+      from_port: 80
+      to_port: 80
+      cidr_ip: '0.0.0.0/0'
+    - proto: tcp
+      from_port: 8080
+      to_port: 8080
```

Comment:
You should be able to drop this rule.
Nginx runs on the jenkins boxes and automatically routes port `80` externally -> `8080` locally.